examples/quickstart-dev.html

```diff
@@ -2,22 +2,25 @@
 
 <head>
   <meta charset="utf-8" />
-  <title>Esri Leaflet Vector Quickstart</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Esri Leaflet Vector Quickstart (DEV TEST)</title>
 
   <!-- Load Leaflet from CDN -->
   <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.0/dist/leaflet.css" />
   <script src="https://unpkg.com/leaflet@1.5.0/dist/leaflet.js"></script>
 
   <!--
-    Load mapbox-gl from CDN for dev/debug purposes because it is not bundled in Esri Leaflet Vector's dev/debug code
+    Load mapbox-gl from CDN or local node_modules for dev/debug purposes because it is not bundled in Esri Leaflet Vector's dev/debug code
     (note also that loading mapbox-gl.css is not necessary)
   -->
-  <script src="https://unpkg.com/mapbox-gl@1.4.0/dist/mapbox-gl.js"></script>
+  <script src="../node_modules/mapbox-gl/dist/mapbox-gl.js"></script>
 
-  <!-- Esri Leaflet and Esri Leaflet Vector plugin dev/debug version -->
-  <script src="https://unpkg.com/esri-leaflet@3/dist/esri-leaflet.js"></script>
+  <!-- Load Esri Leaflet and Esri Leaflet Vector plugin dev/debug version -->
+  <script src="https://unpkg.com/esri-leaflet@2.3.0/dist/esri-leaflet.js"></script>
```

Comment:
Should this stay at v3?
```suggestion
  <script src="https://unpkg.com/esri-leaflet@3/dist/esri-leaflet.js"></script>
```

Reply:
Good point, I wasn't sure how we'd want to deal with these. Should they approximate what is in package.json? Should we just point to local node_modules installations instead of CDN?

Reply:
local node_modules install feels better actually, if that's possible